umbraco-heartcore/getting-started/graphql-playground.md

```diff
@@ -17,31 +17,27 @@ The Playground comes with basic features such as real-time error highlighting, s
 1. Log into the project backoffice
 2. Navigate to the Settings dashboard
 3. Find the Headless tree near the bottom of the navigation list
-4. Click on GraphQL Playground
+4. Click on GraphQL
 
-![Where to find GraphQL Playground](images/where-to-find-graphql-playground.png)
+![Where to find GraphQL](images/where-to-find-graphql-playground.png)
 
 From there, you can start querying as soon as your content is published.
 
 ## Features
 
-### Documentation
+### Documentation & Schema
 
-The GraphQL Playground's built-in documentation can be found on the right side, under the "Docs" tab.
+The GraphQL Playground's build-in documentation and schema browser can be found on the left toolbar
 
-The documentation tab grants a quick overview of how content is structured using the Document Types in your solution. It also gives an idea of how to access specific properties on each of the Document Types as well as nested properties and properties from Compositions.
+This button will open a pane that grants you a quick overview of how content is structured using the Document Types in your solution.
 
-![Documentation Tab in the Playground](images/docs-helper.png)
+It also gives an idea of how to access specific properties on each of the Document Types as well as nested properties and properties from Compositions.
 
-The documentation will be automatically updated as you add new Document Types to match the schema you have on your project.
+While at the same time also explains relationships between specific object and properties in your solution.
 
-## Schema
+This will be updated automatically as you make change to the projects schema meaning Document Types, Data Types, Document Type Compositions and Document Type Elements.
```

Comment:
```suggestion
This will be updated automatically as you make changes to the project's schema meaning Document Types, Data Types, Document Type Compositions, and Document Type Elements.
```